### ISSUE TITLE
similarity matrix calculation issue in hierverb.py

### ISSUE THREAD

**Issue by JaleelRadhu** [DreamH1gh/TACL2024#5]
````markdown
### Possible Bug in `laod_label_sim` Function in DCL/models/hierverb.py

In the function:

```python
def laod_label_sim(self, path):
    label_des = json.load(open(path))
    emb = torch.zeros(len(label_des), 768)
    for l in label_des:
        output = self.tokenizer(l, max_length=512, add_special_tokens=True, truncation=True, padding=True, return_tensors="pt")
        emb[l] += output.pooled_output
    return torch.cdist(emb, emb)
````

Shouldn't the argument to `self.tokenizer` be `label_des[l]` instead of `l`?

It seems like we are trying to compute a **similarity matrix among the label name**, not their label descriptions. Using `l` would tokenize the key (label name) rather than the actual description text (`label_des[l]`).
